tests/dbgp/dbgpclient.php

```diff
@@ -60,8 +93,7 @@ function doRead( $conn )
 			$read = preg_replace( '@(2002-20[0-9]{2})@', '2002-2099', $read );
 			echo $read, "\n\n";
 
-			if ( preg_match( '@<stream xmlns="urn.debugger_protocol_v1" xmlns:xdebug@', $read ) )
-			{
+			if ( preg_match( '@<stream xmlns="urn.debugger_protocol_v1" xmlns:xdebug@', $read ) ) {
```

Comment:
No problems, I understand. It should be changed back now to what it was. I always want everything to "follow the rulebook" so to speak and if I see an error I like to fix it ;-)